doc/manual/management_system.rst

```diff
@@ -32,31 +36,66 @@ The :ref:`dashboard <frontend-artiq-dashboard>` connects to the master and is th
 
 The dashboard remembers and restores GUI state (window/dock positions, last values entered by the user, etc.) in between instances. This information is stored in a file called ``artiq_dashboard_{server}_{port}.pyon`` in the configuration directory (e.g. generally ``~/.config/artiq`` for Unix, same as data directory for Windows), distinguished in subfolders by ARTIQ version.
 
+.. note::
+
+   To find where the configuration files are stored on your machine, try the command: ::
+
+      python -c "from artiq.tools import get_user_config_dir; print(get_user_config_dir())"
+
 Browser
 ^^^^^^^
 
-The :ref:`browser <frontend-artiq-browser>` is used to read ARTIQ ``results`` HDF5 files and run experiment :meth:`~artiq.language.environment.Experiment.analyze` functions, in particular to retrieve previous result databases, process them, and display them in ARTIQ applets. The browser also remembers and restores its GUI state; this is stored in a file called simply ``artiq_browser``, kept in the same configuration directory as the dashboard.
+The :ref:`browser <frontend-artiq-browser>` is used to read ARTIQ ``results`` HDF5 files and run experiment :meth:`~artiq.language.environment.Experiment.analyze` functions, in particular to retrieve previous result databases, process them, and display them in ARTIQ applets. The browser also remembers and restores its GUI state; this is stored in a file called simply ``artiq_browser.pyon``, kept in the same configuration directory as the dashboard.
+
+The browser *can* connect to the master, specifically in order to be able to access the master's store of datasets and to upload new datasets to it, but alone among the management system components it can also be run completely standalone. It does not require a connected master to run. However, it requires filesystem access to the ``results`` files to be of much use.
```

Comment:
"along among" is not strictly accurate since the proxies can *technically* be run standalone (ofc they almost never are). It may also change in the future should more components / helpers be introduced. I think the sentence is clear enough without adding this extra detail.